terraform/bigquery.tf

```
@@ -505,6 +505,7 @@ module "metrics_views" {
 
   dataset_id    = google_bigquery_dataset.default.dataset_id
   base_table_id = google_bigquery_table.unique_events_view.table_id
+  base_tvf_id   = google_bigquery_routine.unique_events_by_date_type.routine_id
```

```suggestion
  base_tvf_id   = google_bigquery_routine.pull_request_events_by_date.routine_id
```

This should be either `pull_request_events_by_date` or `pull_requests_by_date`.